pom.xml

```diff
@@ -89,6 +92,13 @@
             <groupId>org.jdbi</groupId>
             <artifactId>jdbi</artifactId>
             <version>${jdbi.version}</version>
+            <scope>provided</scope>
```

Comment:
I changed the scope to "provided", assuming the import of jdbi is done explicitly by the user of this library, and not transitively through the import of the opentracing for jdbi.
This way we make sure that users of jdbi v2 will not get the jdbi3 imported to their packages unwillingly, and vice-versa.